src/applications/timeouts.ts

```diff
@@ -173,5 +209,5 @@ export function ensureValidAppTimeouts(timeouts) {
     );
   }
 
-  return result;
+  return result as AppOrParcelTimeouts;
```

Comment:
This type cast is safe, since globalTimeoutConfig is guaranteed to be a valid AppOrParcelTimeouts object, `result` is guaranteed to be so as well